rincon-orlando/academy-go-q42021
# Deliverable2

How to test:
Following APIs are available:

To get all Pokemons:
```GET http://localhost:8082/pokemons```

To get just one of them:
```
GET http://localhost:8082/pokemons/1
GET http://localhost:8082/pokemons/2
GET http://localhost:8082/pokemons/3
```

To fetch Pokemons from the external PokemonAPI (and persist result in the CSV file)
```GET http://localhost:8082/pokemons/fetch```

Covers:
- Create a client to consume an external API
- Add an endpoint to consume the external API client
- The information obtained should be stored in the CSV file
- Update the endpoint made in the first deliverable to display the result as a JSON (it was already that way)
- Refactor if needed _(this was the major change I did trying to refactor as much as possible for clean architecture and testability)_

Does not cover yet:
- Add unit testing
